src/components/ComeBackFeedback/ComeBackFeedback.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import SolidButton from '../SolidButton/SolidButton';
+import feedback from '../../img/feedback.svg';
+import comeBackLater from '../../img/comeBackLater.svg';
+import './ComeBackFeedback.css';
+
+const ComeBackFeedback = () => {
+    return (
+        <div className="comeback-feedback">
+
```

Comment:
this is really minor but it's considered standard practice to not include extra lines in HTML. if you're using VSCode, then you can install a really neat extension called Prettier that will automatically format your code!